.travis.yml

```diff
@@ -1,21 +1,36 @@
+os:
+  - linux
+  - windows
 language: go
 go:
-- 1.14.x
-- 1.13.x
-
+  - 1.13.x
+  - 1.14.x
+git:
+  depth: false
+env:
+  - V=
+  - V=v2
 install:
-- go get -t ./...
-- go get github.com/mattn/goveralls
-- go get github.com/wadey/gocovmerge
-- go get -u honnef.co/go/tools/cmd/staticcheck
-- go get -u github.com/client9/misspell/cmd/misspell
-
+  - go get -t ./...
+  - go get -u honnef.co/go/tools/cmd/staticcheck
+  - go get -u github.com/client9/misspell/cmd/misspell
+  - go get github.com/mattn/goveralls
+  - go get github.com/wadey/gocovmerge
 before_script:
-- $(exit $(go fmt ./... | wc -l))
-- go vet ./...
-- misspell -error -locale US .
-- staticcheck ./...
-
+  - cd $TRAVIS_BUILD_DIR/${V}
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then EXCLUDE_VENDOR=$(go list ./... | grep -v "/vendor/") && $(exit $(go fmt $EXCLUDE_VENDOR | wc -l)) && go vet $EXCLUDE_VENDOR; fi
+  - go vet ./...
+  - misspell -error -locale US .
+  - staticcheck ./...
 script:
-- go test -v -race ./...
-- if [[ "$TRAVIS_GO_VERSION" =~ 1.13 ]]; then ./scripts/cov.sh TRAVIS; fi
+  - go test -v -race ./...
+  - go test -v -coverprofile=./coverage.out ./...
+deploy:
+- provider: script
+  script: $HOME/gopath/bin/goveralls -coverprofile=v2/coverage.out -service travis-ci
```

Comment:
Not sure about `v2/coverage.out`. Is that where the file will be? We call `go test -v -coverprofile=./coverage.out ./...`, does that work by creating a coverage.out in the `/v2` directory?

Reply:
Sure it does. When we activate the v2 release, then I’ll have that coverage be posted.